jena-rdfconnection/src/test/java/org/apache/jena/rdfconnection/TestRDFConnectionHTTP.java

```diff
@@ -1,99 +0,0 @@
-/*
```

Comment:
Where did the tests from this class go to?
(It should have been added to `TS_RDFConnection` when added.)

Reply:
These tests used the `getAppProvidedAcceptHeader` method, which I introduced only for testing in a previous PR and which was also stated in the javadoc ("For testing only."). In hindsight, a better solution would have been a package private method together with the test case in the `org.apache.jena.sparql.exec.http` package within jena-rdfconnection :/

The field and the method are now marked as deprecated.

The value of the `appProvidedAcceptHeader` used to rely on the QueryType - but since this wouldn't work generically for strings, this PR removed this logic. The header is now chosen based on the exec call such as `.ask()`.